src/server/models/tasks.js

```diff
@@ -5,11 +5,21 @@ const ObjectId = Schema.ObjectId;
 
 const TaskSchema = new Schema({
   id: ObjectId,
-  name: String,
-  body: String,
-  date: Date,
+  name: {
+    type: String,
+    required: true,
+  },
+  body: {
+    type: String,
+    required: true,
+  },
+  date: {
```

Comment:
Why only one date? A task should have a start & end date/time.

Reply:
Discuss this with @Shirley-Liz

Reply:
I will see to it that i discuss it with Shirley

Reply:
i have thought about it and i think we can use  the start and end dates

Reply:
We have thought about it and i think we can use the start_date,end_date and the time.You can view them in models/tasks.js